test/utils/sequence.ts

```diff
@@ -262,6 +277,17 @@ export function printTopology(topology: ConfigTopology, threshold?: ethers.BigNu
     return [`subdigest: ${topology.subdigest}`]
   }
 
+  if (isNestedLeaf(topology)) {
+    const result: string[] = [`internalThreshold: ${topology.internalThreshold} - externalWeight: ${topology.externalWeight}`]
+    const signers = printTopology(topology.tree, undefined, inverse)
+    for (let i = 0; i < signers.length; i++) {
+      const prefix = i === 0 ? '└─ ' : '  '
```

Comment:
Small question, is the prefix supposed to be the same length regardless of the signer index?